### PR TITLE
project finished

### PR DESCRIPTION
**Pull request by Aapje185**

### CODE REVIEW

File: MathGame.Aapje185/UI/UserInterface.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    public void GetOptionFromUser()
+    {
+        int userChoice;
+        while (!TryParse(Console.ReadLine(), out userChoice) || !_options.ContainsKey(userChoice))
+        {
+            Console.WriteLine($"The chosen option is not available, please enter a valid option.");
+        }
+
+        if (userChoice == 5) return;
```
Comment (TheCSharpAcademy) [inline MathGame.Aapje185/UI/UserInterface.cs:46]: I'm getting this error when I choose division 

image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@Aapje185 Thanks for submitting! It's looking great. Could you please just fix the issue in the comment? 😊

File: MathGame.Aapje185/UI/UserInterface.cs (hunk 1 above)
Comment (Aapje185) [reply MathGame.Aapje185/UI/UserInterface.cs]: Thank you for reviewing. Bug should be fixed now.

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@Aapje185 almost there! As per second requirement, you shouldn't show equations that result in non-integers: 
image

image

**Comment by TheCSharpAcademy** [comment]
@Aapje185 I'm closing the PR as it's been 2 weeks, but feel free to reopen it if you want to continue 😊

**Pull request closed without merging**